src/Display/Gallery/Gallery.test.tsx

```diff
@@ -68,6 +74,14 @@ describe('<Gallery /> rendering', () => {
     );
     expect(asFragment()).toMatchSnapshot();
   });
+
+  it('should return console error when item is invalid children', () => {
```

Comment:
Nice, thanks for adding this test. Can we add one for rendering GlintsPicture? Or maybe we could also add a storybook too